### ISSUE TITLE
MoP Legendary Quest

### ISSUE THREAD

**Issue by cybrann** [MoPSource/MopCore547#50]
Here will be a list of working and not working quests for Legendary Cloak.

Chapter I:
 - [quest=31488]Stranger in a Strange Land

**Comment by Crypticaz** [comment]
Could you specify what's working and not working? It's just a list of quests

**Comment by cybrann** [comment]
I will, I just listed it until I test it.

**Comment by cybrann** [comment]
I have updated the list with what is working and what is not working.

**Comment by cooler-SAI** [comment]
only one?

**Comment by alexkulya** [comment]
Not all at once. Over time, recovered.

**Comment by cybrann** [comment]
Yes, currently only one is working. Also I needed to first fixup the quest order, it was extremely messed up

**Comment by doverman930** [comment]
dejenme eso a mi en mi repack funciona correctamente la linea de quest legendarias yo ya lo arregle
